Update AuthenticationMethodsPolicy

jpettere opened the pull request:
Add public preview changes

Schema PR: 7082640

opbld33 commented:
Learn Build status updates of commit _[8a6e864](https://github.com/microsoftgraph/microsoft-graph-docs/commits/8a6e8648f24871372e095d686438ec93fc56c36d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[changelog/Microsoft.AuthenticationMethodsPolicy.json](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/changelog/Microsoft.AuthenticationMethodsPolicy.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/77e68b91-cf14-3558-43ca-0d6d735c492d/202211152010276529-19255/BuildReport?accessString=ec2d9bcf2929d9c77aac98de58d5f489e35f90c60fa861e8effd81d71a0d2be5).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

opbld33 commented:
Learn Build status updates of commit _[b5dae1a](https://github.com/microsoftgraph/microsoft-graph-docs/commits/b5dae1a862955d9df9c397172360102c36d925fc)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/emailauthenticationmethodconfiguration-delete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/emailauthenticationmethodconfiguration-delete.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/emailauthenticationmethodconfiguration-delete?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/emailauthenticationmethodconfiguration-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/emailauthenticationmethodconfiguration-get.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/emailauthenticationmethodconfiguration-get?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/emailauthenticationmethodconfiguration-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/emailauthenticationmethodconfiguration-update.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/emailauthenticationmethodconfiguration-update?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/fido2authenticationmethodconfiguration-delete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/fido2authenticationmethodconfiguration-delete.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/fido2authenticationmethodconfiguration-delete?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/fido2authenticationmethodconfiguration-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/fido2authenticationmethodconfiguration-get.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/fido2authenticationmethodconfiguration-get?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/fido2authenticationmethodconfiguration-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/fido2authenticationmethodconfiguration-update.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/fido2authenticationmethodconfiguration-update?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/microsoftauthenticatorauthenticationmethodconfiguration-delete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/microsoftauthenticatorauthenticationmethodconfiguration-delete.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/microsoftauthenticatorauthenticationmethodconfiguration-delete?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/microsoftauthenticatorauthenticationmethodconfiguration-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/microsoftauthenticatorauthenticationmethodconfiguration-get.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/microsoftauthenticatorauthenticationmethodconfiguration-get?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/microsoftauthenticatorauthenticationmethodconfiguration-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/microsoftauthenticatorauthenticationmethodconfiguration-update.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/microsoftauthenticatorauthenticationmethodconfiguration-update?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/smsauthenticationmethodconfiguration-delete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/smsauthenticationmethodconfiguration-delete.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/smsauthenticationmethodconfiguration-delete?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/smsauthenticationmethodconfiguration-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/smsauthenticationmethodconfiguration-get.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/smsauthenticationmethodconfiguration-get?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/smsauthenticationmethodconfiguration-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/smsauthenticationmethodconfiguration-update.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/smsauthenticationmethodconfiguration-update?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/softwareoathauthenticationmethodconfiguration-delete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/softwareoathauthenticationmethodconfiguration-delete.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/softwareoathauthenticationmethodconfiguration-delete?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/softwareoathauthenticationmethodconfiguration-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/softwareoathauthenticationmethodconfiguration-get.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/softwareoathauthenticationmethodconfiguration-get?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/softwareoathauthenticationmethodconfiguration-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/softwareoathauthenticationmethodconfiguration-update.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/softwareoathauthenticationmethodconfiguration-update?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/temporaryaccesspassauthenticationmethodconfiguration-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/temporaryaccesspassauthenticationmethodconfiguration-get.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/temporaryaccesspassauthenticationmethodconfiguration-get?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/voiceauthenticationmethodconfiguration-delete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/voiceauthenticationmethodconfiguration-delete.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/voiceauthenticationmethodconfiguration-delete?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/voiceauthenticationmethodconfiguration-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/voiceauthenticationmethodconfiguration-get.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/voiceauthenticationmethodconfiguration-get?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/voiceauthenticationmethodconfiguration-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/voiceauthenticationmethodconfiguration-update.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/voiceauthenticationmethodconfiguration-update?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/x509certificateauthenticationmethodconfiguration-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/x509certificateauthenticationmethodconfiguration-get.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/x509certificateauthenticationmethodconfiguration-get?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/x509certificateauthenticationmethodconfiguration-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/x509certificateauthenticationmethodconfiguration-update.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/x509certificateauthenticationmethodconfiguration-update?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/resources/authenticationmethodconfiguration.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/resources/authenticationmethodconfiguration.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/resources/authenticationmethodconfiguration?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/resources/authenticationmethodspolicy.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/resources/authenticationmethodspolicy.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/resources/authenticationmethodspolicy?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/resources/emailauthenticationmethodconfiguration.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/resources/emailauthenticationmethodconfiguration.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/resources/emailauthenticationmethodconfiguration?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/resources/fido2authenticationmethodconfiguration.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/resources/fido2authenticationmethodconfiguration.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/resources/fido2authenticationmethodconfiguration?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |

This comment lists only the first 25 files in the pull request.
For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/77e68b91-cf14-3558-43ca-0d6d735c492d/202211290235458307-19255/BuildReport?accessString=fe2d755a7779bef6fb2faaffe64f912945c5abad9a22532a44e274d9306ad4a4).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

opbld32 commented:
Learn Build status updates of commit _[0f99b85](https://github.com/microsoftgraph/microsoft-graph-docs/commits/0f99b85f98e331e35c9238bd24dd08c93d87347e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/emailauthenticationmethodconfiguration-delete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/emailauthenticationmethodconfiguration-delete.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/emailauthenticationmethodconfiguration-delete?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/emailauthenticationmethodconfiguration-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/emailauthenticationmethodconfiguration-get.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/emailauthenticationmethodconfiguration-get?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/emailauthenticationmethodconfiguration-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/emailauthenticationmethodconfiguration-update.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/emailauthenticationmethodconfiguration-update?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/fido2authenticationmethodconfiguration-delete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/fido2authenticationmethodconfiguration-delete.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/fido2authenticationmethodconfiguration-delete?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/fido2authenticationmethodconfiguration-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/fido2authenticationmethodconfiguration-get.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/fido2authenticationmethodconfiguration-get?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/fido2authenticationmethodconfiguration-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/fido2authenticationmethodconfiguration-update.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/fido2authenticationmethodconfiguration-update?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/microsoftauthenticatorauthenticationmethodconfiguration-delete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/microsoftauthenticatorauthenticationmethodconfiguration-delete.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/microsoftauthenticatorauthenticationmethodconfiguration-delete?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/microsoftauthenticatorauthenticationmethodconfiguration-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/microsoftauthenticatorauthenticationmethodconfiguration-get.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/microsoftauthenticatorauthenticationmethodconfiguration-get?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/microsoftauthenticatorauthenticationmethodconfiguration-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/microsoftauthenticatorauthenticationmethodconfiguration-update.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/microsoftauthenticatorauthenticationmethodconfiguration-update?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/smsauthenticationmethodconfiguration-delete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/smsauthenticationmethodconfiguration-delete.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/smsauthenticationmethodconfiguration-delete?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/smsauthenticationmethodconfiguration-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/smsauthenticationmethodconfiguration-get.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/smsauthenticationmethodconfiguration-get?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/smsauthenticationmethodconfiguration-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/smsauthenticationmethodconfiguration-update.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/smsauthenticationmethodconfiguration-update?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/softwareoathauthenticationmethodconfiguration-delete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/softwareoathauthenticationmethodconfiguration-delete.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/softwareoathauthenticationmethodconfiguration-delete?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/softwareoathauthenticationmethodconfiguration-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/softwareoathauthenticationmethodconfiguration-get.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/softwareoathauthenticationmethodconfiguration-get?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/softwareoathauthenticationmethodconfiguration-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/softwareoathauthenticationmethodconfiguration-update.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/softwareoathauthenticationmethodconfiguration-update?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/temporaryaccesspassauthenticationmethodconfiguration-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/temporaryaccesspassauthenticationmethodconfiguration-get.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/temporaryaccesspassauthenticationmethodconfiguration-get?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/voiceauthenticationmethodconfiguration-delete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/voiceauthenticationmethodconfiguration-delete.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/voiceauthenticationmethodconfiguration-delete?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/voiceauthenticationmethodconfiguration-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/voiceauthenticationmethodconfiguration-get.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/voiceauthenticationmethodconfiguration-get?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/voiceauthenticationmethodconfiguration-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/voiceauthenticationmethodconfiguration-update.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/voiceauthenticationmethodconfiguration-update?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/x509certificateauthenticationmethodconfiguration-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/x509certificateauthenticationmethodconfiguration-get.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/x509certificateauthenticationmethodconfiguration-get?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/x509certificateauthenticationmethodconfiguration-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/x509certificateauthenticationmethodconfiguration-update.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/x509certificateauthenticationmethodconfiguration-update?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/resources/authenticationmethodconfiguration.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/resources/authenticationmethodconfiguration.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/resources/authenticationmethodconfiguration?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/resources/authenticationmethodspolicy.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/resources/authenticationmethodspolicy.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/resources/authenticationmethodspolicy?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/resources/emailauthenticationmethodconfiguration.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/resources/emailauthenticationmethodconfiguration.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/resources/emailauthenticationmethodconfiguration?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/resources/fido2authenticationmethodconfiguration.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/resources/fido2authenticationmethodconfiguration.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/resources/fido2authenticationmethodconfiguration?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |

This comment lists only the first 25 files in the pull request.
For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/77e68b91-cf14-3558-43ca-0d6d735c492d/202211290238027920-19255/BuildReport?accessString=76c92d33703c1da4065a7bcbbb6d1be086c081afd275e9bf0c5bfdbdc7998d76).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

opbld32 commented:
Learn Build status updates of commit _[80fabd6](https://github.com/microsoftgraph/microsoft-graph-docs/commits/80fabd60779b5830a15470eaace0d4bdc9ca84ac)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/emailauthenticationmethodconfiguration-delete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/emailauthenticationmethodconfiguration-delete.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/emailauthenticationmethodconfiguration-delete?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/emailauthenticationmethodconfiguration-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/emailauthenticationmethodconfiguration-get.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/emailauthenticationmethodconfiguration-get?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/emailauthenticationmethodconfiguration-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/emailauthenticationmethodconfiguration-update.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/emailauthenticationmethodconfiguration-update?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/fido2authenticationmethodconfiguration-delete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/fido2authenticationmethodconfiguration-delete.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/fido2authenticationmethodconfiguration-delete?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/fido2authenticationmethodconfiguration-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/fido2authenticationmethodconfiguration-get.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/fido2authenticationmethodconfiguration-get?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/fido2authenticationmethodconfiguration-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/fido2authenticationmethodconfiguration-update.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/fido2authenticationmethodconfiguration-update?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/microsoftauthenticatorauthenticationmethodconfiguration-delete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/microsoftauthenticatorauthenticationmethodconfiguration-delete.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/microsoftauthenticatorauthenticationmethodconfiguration-delete?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/microsoftauthenticatorauthenticationmethodconfiguration-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/microsoftauthenticatorauthenticationmethodconfiguration-get.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/microsoftauthenticatorauthenticationmethodconfiguration-get?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/microsoftauthenticatorauthenticationmethodconfiguration-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/microsoftauthenticatorauthenticationmethodconfiguration-update.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/microsoftauthenticatorauthenticationmethodconfiguration-update?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/smsauthenticationmethodconfiguration-delete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/smsauthenticationmethodconfiguration-delete.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/smsauthenticationmethodconfiguration-delete?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/smsauthenticationmethodconfiguration-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/smsauthenticationmethodconfiguration-get.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/smsauthenticationmethodconfiguration-get?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/smsauthenticationmethodconfiguration-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/smsauthenticationmethodconfiguration-update.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/smsauthenticationmethodconfiguration-update?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/softwareoathauthenticationmethodconfiguration-delete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/softwareoathauthenticationmethodconfiguration-delete.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/softwareoathauthenticationmethodconfiguration-delete?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/softwareoathauthenticationmethodconfiguration-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/softwareoathauthenticationmethodconfiguration-get.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/softwareoathauthenticationmethodconfiguration-get?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/softwareoathauthenticationmethodconfiguration-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/softwareoathauthenticationmethodconfiguration-update.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/softwareoathauthenticationmethodconfiguration-update?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/temporaryaccesspassauthenticationmethodconfiguration-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/temporaryaccesspassauthenticationmethodconfiguration-get.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/temporaryaccesspassauthenticationmethodconfiguration-get?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/voiceauthenticationmethodconfiguration-delete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/voiceauthenticationmethodconfiguration-delete.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/voiceauthenticationmethodconfiguration-delete?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/voiceauthenticationmethodconfiguration-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/voiceauthenticationmethodconfiguration-get.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/voiceauthenticationmethodconfiguration-get?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/voiceauthenticationmethodconfiguration-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/voiceauthenticationmethodconfiguration-update.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/voiceauthenticationmethodconfiguration-update?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/x509certificateauthenticationmethodconfiguration-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/x509certificateauthenticationmethodconfiguration-get.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/x509certificateauthenticationmethodconfiguration-get?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/api/x509certificateauthenticationmethodconfiguration-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/api/x509certificateauthenticationmethodconfiguration-update.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/x509certificateauthenticationmethodconfiguration-update?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/resources/authenticationmethodconfiguration.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/resources/authenticationmethodconfiguration.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/resources/authenticationmethodconfiguration?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/resources/authenticationmethodspolicy.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/resources/authenticationmethodspolicy.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/resources/authenticationmethodspolicy?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/resources/emailauthenticationmethodconfiguration.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/resources/emailauthenticationmethodconfiguration.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/resources/emailauthenticationmethodconfiguration?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |
[api-reference/beta/resources/fido2authenticationmethodconfiguration.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2022-11-16-jupetter-authmethodspolicy/api-reference/beta/resources/fido2authenticationmethodconfiguration.md) | :white_check_mark:Succeeded | [View](https://review.learn.microsoft.com/en-us/graph/api/resources/fido2authenticationmethodconfiguration?view=graph-rest-beta&branch=pr-en-us-19255) (graph-rest-beta) |

This comment lists only the first 25 files in the pull request.
For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/77e68b91-cf14-3558-43ca-0d6d735c492d/202211290254324605-19255/BuildReport?accessString=07096aeff48a8b82dd8b14ecd8f460349b3c82c7c70776444d0a8ef2bc353ff0).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

FaithOmbongi commented on api-reference/beta/api/softwareoathauthenticationmethodconfiguration-get.md:
```diff
@@ -0,0 +1,100 @@
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Read the properties and relationships of a [softwareOathAuthenticationMethodConfiguration](../resources/softwareoathauthenticationmethodconfiguration.md) object, which represents the Third-Party Software OATH authentication method policy for the Azure AD tenant.
```
```suggestion
Read the properties and relationships of a [softwareOathAuthenticationMethodConfiguration](../resources/softwareoathauthenticationmethodconfiguration.md) object, which represents the third-party software OATH authentication method policy for the Azure AD tenant.
```

FaithOmbongi commented on api-reference/beta/api/softwareoathauthenticationmethodconfiguration-get.md:
````diff
@@ -0,0 +1,100 @@
+```
+
+## Optional query parameters
+This method supports some of the OData query parameters to help customize the response. For general information, see [OData query parameters](/graph/query-parameters).
````
Mention specific query parameters; If none, say ".. method does not support OData query ...."

FaithOmbongi commented on api-reference/beta/api/softwareoathauthenticationmethodconfiguration-delete.md:
```diff
@@ -0,0 +1,72 @@
+|:---|:---|
+|Delegated (work or school account)|Policy.ReadWrite.AuthenticationMethod|
+|Delegated (personal Microsoft account)|Not supported.|
+|Application|Policy.ReadWrite.AuthenticationMethod|
```
Call our RBAC

jpettere replied:
Not sure what you mean by this comment.

FaithOmbongi commented on api-reference/beta/api/softwareoathauthenticationmethodconfiguration-update.md:
```diff
@@ -0,0 +1,89 @@
+
+### Response
+The following is an example of the response
+>**Note:** The response object shown here might be shortened for readability.
```
nit
```suggestion
```

FaithOmbongi commented on api-reference/beta/api/smsauthenticationmethodconfiguration-update.md:
```diff
@@ -110,21 +110,14 @@ Content-Type: application/json
 ### Response
-**Note:** The response object shown here might be shortened for readability.
+The following is an example of the response
+>**Note:** The response object shown here might be shortened for readability.
```
nit
```suggestion
```

FaithOmbongi commented on api-reference/beta/resources/voiceauthenticationmethodconfiguration.md:
```diff
@@ -0,0 +1,64 @@
+---
+title: "voiceAuthenticationMethodConfiguration resource type"
+description: "Represents a Phone Call authenticaiton methods policy"
```
```suggestion
description: "Represents a phone call authentication methods policy."
```

FaithOmbongi commented on api-reference/beta/resources/voiceauthenticationmethodconfiguration.md:
```diff
@@ -0,0 +1,64 @@
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents a Phone Call authentication methods policy. Authentication methods policies define configuration settings and users or groups that are enabled to use the authentication method.
```
```suggestion
Represents a phone call authentication methods policy. Authentication methods policies define configuration settings and users or groups that are enabled to use the authentication method.
```

FaithOmbongi commented on api-reference/beta/resources/voiceauthenticationmethodconfiguration.md:
```diff
@@ -0,0 +1,64 @@
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|excludeTargets|[excludeTarget](../resources/excludetarget.md) collection|A collection of users or groups of users that are excluded from a policy.|
```
Same comment about groups only, no individual users

FaithOmbongi commented on api-reference/beta/resources/voiceauthenticationmethodconfiguration.md:
```diff
@@ -0,0 +1,64 @@
+|:---|:---|:---|
+|excludeTargets|[excludeTarget](../resources/excludetarget.md) collection|A collection of users or groups of users that are excluded from a policy.|
+|id|String|The authentication method policy identifier.|
+|isOfficePhoneAllowed|Boolean|`true` if users can register office phones, otherwise, `false`|
```
```suggestion
|isOfficePhoneAllowed|Boolean|`true` if users can register office phones, otherwise, `false`. |
```

FaithOmbongi commented on api-reference/beta/resources/voiceauthenticationmethodconfiguration.md:
```diff
@@ -0,0 +1,64 @@
+## Relationships
+|Relationship|Type|Description|
+|:---|:---|:---|
+|includeTargets|[voiceAuthenticationMethodTarget](../resources/voiceauthenticationmethodtarget.md) collection|A collection of users or groups who are enabled to use the authentication method. Expanded by default.|
```
Same comment about groups or all users only. No individual users.

FaithOmbongi commented on api-reference/beta/api/voiceauthenticationmethodconfiguration-delete.md:
```diff
@@ -0,0 +1,77 @@
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Remove changes made to the [Phone Call authentication method policy](../resources/voiceauthenticationmethodconfiguration.md) by reverting the policy to its default configuration.
```
```suggestion
Revert the [phone call authentication method policy](../resources/voiceauthenticationmethodconfiguration.md) b to its default configuration.
```

FaithOmbongi commented on api-reference/beta/api/voiceauthenticationmethodconfiguration-delete.md:
```diff
@@ -0,0 +1,77 @@
+
+### Response
+The following is an example of the response
+>**Note:** The response object shown here might be shortened for readability.
```
nit
```suggestion
```

FaithOmbongi commented on api-reference/beta/api/voiceauthenticationmethodconfiguration-get.md:
````diff
@@ -0,0 +1,101 @@
+```
+
+## Optional query parameters
+This method supports some of the OData query parameters to help customize the response. For general information, see [OData query parameters](/graph/query-parameters).
````
Call out the supported query parameters; If none, say "... method does not support OData query .."

FaithOmbongi commented on api-reference/beta/api/voiceauthenticationmethodconfiguration-update.md:
```diff
@@ -0,0 +1,89 @@
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Update the properties of a [voiceAuthenticationMethodConfiguration](../resources/voiceauthenticationmethodconfiguration.md) object, which represents the Phone Call authentication method policy for the Azure AD tenant.
```
```suggestion
Update the properties of a [voiceAuthenticationMethodConfiguration](../resources/voiceauthenticationmethodconfiguration.md) object, which represents the phone call authentication method policy for the Azure AD tenant.
```

FaithOmbongi commented on api-reference/beta/api/voiceauthenticationmethodconfiguration-update.md:
```diff
@@ -0,0 +1,89 @@
+
+### Response
+The following is an example of the response
+>**Note:** The response object shown here might be shortened for readability.
```
nit
```suggestion
```

FaithOmbongi commented on api-reference/beta/api/softwareoathauthenticationmethodconfiguration-update.md:
```diff
@@ -0,0 +1,89 @@
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Update the properties of a [softwareOathAuthenticationMethodConfiguration](../resources/softwareoathauthenticationmethodconfiguration.md) object, which represents the Third-Party Software OATH authentication method policy for the Azure AD tenant.
```
```suggestion
Update the properties of a [softwareOathAuthenticationMethodConfiguration](../resources/softwareoathauthenticationmethodconfiguration.md) object, which represents the third-party software OATH authentication method policy for the Azure AD tenant.
```

FaithOmbongi commented on api-reference/beta/resources/smsauthenticationmethodtarget.md:
```diff
@@ -20,7 +20,7 @@ A collection of users or groups enabled to use [Text Message authentication meth
 |id|String|Object ID of an Azure AD user or group.|
 |isRegistrationRequired|Boolean|Determines whether the user is enforced to register the authentication method. **Not supported**.|
 |isUsableForSignIn|Boolean|Determines if the users or groups can use this authentication method to sign in to Azure AD. The value is always `true`.|
```
- isUsableForSignIn can now be turned off, I believe - update the reference.
- Indicate what true and false correspond to. I see from your presentation it's:

true - can be used for MFA, SSPR, and first factor
false- can be used for MFA and SSPR only. First-factor not supported.

FaithOmbongi commented on api-reference/beta/resources/voiceauthenticationmethodtarget.md:
```diff
@@ -0,0 +1,46 @@
+---
+title: "voiceAuthenticationMethodTarget resource type"
+description: "A collection of users or groups enabled to use Phone Call authentication methods policy."
```
Same comment about groups or all users, not specific users.
```suggestion
description: "A collection of users or groups enabled to use phone call authentication method via policy."
```

FaithOmbongi commented on api-reference/beta/resources/voiceauthenticationmethodtarget.md:
```diff
@@ -0,0 +1,46 @@
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+A collection of users or groups enabled to use [Phone call authentication methods policy](../resources/voiceAuthenticationMethodConfiguration.md) in Azure AD.
```
Same comment as in the metadata also applies
```suggestion
A collection of users or groups enabled to use phone call authentication via the [phone call authentication methods policy](../resources/voiceAuthenticationMethodConfiguration.md) in Azure AD.
```

FaithOmbongi commented on api-reference/beta/resources/voiceauthenticationmethodtarget.md:
```diff
@@ -0,0 +1,46 @@
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|id|String|Object ID of an Azure AD user or group.|
+|isRegistrationRequired|Boolean|Determines whether the user is enforced to register the authentication method. **Not supported**.|
+|targetType|authenticationMethodTargetType|The possible values are: `user`, `group`, and `unknownFutureValue`. Inherited from [authenticationMethodTarget](authenticationMethodTarget.md).|
```
For id and targetType - My comments about group and now individual users applies.

FaithOmbongi reviewed (changes requested):
Thank you @jpettere for your work on this PR. I've left suggestions inline in addition to the following comments.

_PS: With regard to my suggestions for includeTargets, I'm not sure what get's enforced immediately vs what's in transition until Jan 2024._

1. Changelog - Add the changelog as a standalone object in the file. Don't overwrite an existing changelog object.
2. For the **excludeTargets** property description, I understand that you can only specify groups, not individual users. Please indicate so in the authenticationmethodconfiguration.md and apply that string across all references.
3. **GLOBAL:** For [excludeTarget complex type](https://learn.microsoft.com/en-us/graph/api/resources/excludetarget?view=graph-rest-beta), should **id** therefore only indicate group? Should **targetType** say `user` isn't supported?
4. **GLOBAL:** For **includeTargets** property description - Not in the scope of this PR but I believe you should only specify either all users or specific groups, unlike today where you can specify individual users (correct me if I'm wrong). Should the [authenticationmethodtarget ](https://learn.microsoft.com/en-us/graph/api/resources/authenticationmethodtarget?view=graph-rest-beta) resource type be updated as well? For example: will **id** only be for a group? **targetType** is an enum that includes `user`. Should we say this value is no longer supported?
5. Update the TOC -  Surface the phone call and software oath policies.

jpettere commented:
@FaithOmbongi thanks for your help reviewing. I addressed most of your comments and sent you a question about how to fix the validation issues for the changelog.

RE targeting of groups and not users. We updated the UI to only support targeting new groups (previously targeted users will remain) but have not made the same change at the API layer because its quite difficult to implement. We're ironing out plans to update the API and UI to basically ask customers who previously targeted individual users to "fix" a given method policy if they want to update it and when we do this we'll update the docs. For now, technically documenting that we don't support targeting users in MS Graph is inaccurate, but we could avoid mentioning it given the plans. What do you think?

jpettere commented on api-reference/beta/resources/authenticationmethodspolicy.md:
```diff
@@ -29,6 +29,7 @@ Defines authentication methods and the users that are allowed to use them to sig
 |lastModifiedDateTime|DateTimeOffset|The date and time of the last update to the policy.|
 |policyVersion|String|The version of the policy in use.|
+|reconfirmationInDays|Int32|How many days before the user will be asked to reconfirm their method.|
 |registrationEnforcement|[registrationEnforcement](../resources/registrationenforcement.md)|Enforce registration at sign-in time. This property can be used to remind users to set up targeted authentication methods.|
```
If the two properties are marked as hidden in the schema, shouldn't they be hidden from the public graph schema.

jpettere replied:
Yeah the team expects them to be hidden from the graph schema too. Do we need to talk to someone else about this?

FaithOmbongi replied:
Probably chat with the Graph onboarding team. There must be something engineers aren't doing on their end because this issue doesn't affect all objects.

FaithOmbongi commented on api-reference/beta/api/microsoftauthenticatorauthenticationmethodconfiguration-get.md:
```diff
@@ -118,6 +118,8 @@ Content-Type: application/json
         }
     ],
+    "excludeTargets": [],
+    "isSoftwareOathEnabled": "true",
```
This will resolve the validation warning.
```suggestion
    "isSoftwareOathEnabled": true,
```

FaithOmbongi reviewed (changes requested):
Hi @jpettere - I've left some 5 comments of oversights - one will resolve the validation error.
Also document the authenticationMethodTargetType enumeration in the enums.md file.
I've also pushed a commit to fix some incorrectly autogenerated change entries.

jpettere commented on api-reference/beta/resources/excludetarget.md:
```diff
@@ -18,8 +18,8 @@ Represents the users or groups of users that are excluded from a policy.
-|id|String|The object identifier of an Azure AD user or group.|
-|targetType|authenticationMethodTargetType|The type of the authentication method target. Possible values are: `user`, `group`, `unknownFutureValue`.|
+|id|String|The object identifier of an Azure AD group.|
+|targetType|authenticationMethodTargetType|The type of the authentication method target. Possible values are: `group` and `unknownFutureValue`.|
```
Checked with engineering and it shouldn't be noted. They'll remove it from the schema.

FaithOmbongi replied:
That's interesting. I wonder if it'll be possible to deprecate it given the enumeration type is already in the live schema because it has been used by includeTargets for years.  From my understanding, removing it today would be a breaking change.

FaithOmbongi reviewed (approved):
Hi @jpettere - I'm approving this PR but with one comment, about the authenticationMethodTargetType enum. You've mentioned that the team is removing the `user` member from this enum but that looks suspiciously like a breaking change, which I don't think then API reviewers would encourage.

Once your schema PR is approved and merged, verify what goes live for this enum and if it doesn't match the docs at that time, please make the necessary fixes.

Approved with comments.

isvargasmsft commented on changelog/Microsoft.AuthenticationMethodsPolicy.json:
```diff
@@ -195,6 +195,72 @@
+      "Id": "bacd077e-b2ee-4d2b-be5d-b4ca0a6bcfe4",
+      "Cloud": "Prod",
+      "Version": "beta",
+      "CreatedDateTime": "2022-12-20T00:00:00.1642286Z",
```
```suggestion
      "CreatedDateTime": "2022-12-22T00:00:00.1642286Z",
```
